pkg/app/piped/executor/analysis/metrics_analyzer.go

```diff
@@ -132,6 +132,8 @@ func (a *metricsAnalyzer) analyzeWithThreshold(ctx context.Context) (bool, error
 		From: now.Add(-a.cfg.Interval.Duration()),
 		To:   now,
 	}
+
+	a.logPersister.Infof("[%s] Run query: %q, in range: %s", a.cfg.Query, queryRange)
```

Comment:
Should we add the analyzer id into params?

Reply:
Fixed it!